kafka/conn.py

```diff
@@ -1166,10 +1169,21 @@ def _infer_broker_version_from_api_versions(self, api_versions):
         # in reverse order. As soon as we find one that works, return it
         test_cases = [
             # format (<broker version>, <needed struct>)
+            ((2, 5, 0), DescribeAclsRequest[2]),
```

Comment:
Great job on those versions. I had a hard time finding them, did you copy it from somewhere?))

Reply:
I ran the metadata request on each version of kafka, then plugged those in python and did an incremental diff. I was actually thinking of hardcoding those responses into the codebase as the actual api versions chosen would make more sense